tracer/src/Datadog.Trace/PDBs/SymbolMethodExtensions.cs

```diff
@@ -0,0 +1,51 @@
+// <copyright file="SymbolMethodExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.Vendors.dnlib.DotNet.Pdb.Symbols;
+
+namespace Datadog.Trace.PDBs
```

Comment:
[nitpick] .NET naming conventions suggest using `Pdb` here instead of `PBDs`. Acronyms in namespaces should not use plural, and identifiers (including acronyms over two letters in length) should use PascalCase or camelCase. Two-letter acronyms are a special case, like `System.IO` or `IOStream`.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Reply:
Thanks! Fixed in 3f2b6e85328c73714616d44dd2d0fa77ef3b9908.